packages/next-codemod/transforms/__testfixtures__/next-request-geo-ip/destructure-alias.input.ts

```diff
@@ -0,0 +1,7 @@
+// @ts-nocheck
```

Comment:
do we need this if these files are already exlcude in tsconfig?

Reply:
We can remove it as long as we allow `Cannot find module '@vercel/functions' or its corresponding type declarations.ts(2307)` displayed on our editor, compiling will pass without it.